koku/koku/settings.py

```diff
@@ -389,4 +389,7 @@
 )
 
 # Demo Accounts list
-DEMO_ACCOUNTS = ENVIRONMENT.json("DEMO_ACCOUNTS", {})
+try:
+    DEMO_ACCOUNTS = ENVIRONMENT.json("DEMO_ACCOUNTS", {})
```

Comment:
Did I just mess this up and it should be:
DEMO_ACCOUNTS = ENVIRONMENT.json("DEMO_ACCOUNTS", default={})

Reply:
I also tried the `default={}` there and it still came out a little wonky. I think it is using the built in .json in the dir(Environment), and when the DEMO_ACCOUNT doesn't exit it just throughs a value error.

Reply:
Same with default=“{}”?
Just wondering if it’s gonna parse it and needs a string instead of a dict